app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinPresenter.kt

```diff
@@ -1,20 +1,37 @@
 package org.p2p.wallet.auth.ui.pin.newcreate
 
+import org.p2p.wallet.R
 import org.p2p.wallet.auth.analytics.AdminAnalytics
+import org.p2p.wallet.auth.interactor.AuthInteractor
 import org.p2p.wallet.auth.interactor.AuthLogoutInteractor
+import org.p2p.wallet.auth.interactor.CreateWalletInteractor
+import org.p2p.wallet.auth.model.BiometricStatus
 import org.p2p.wallet.common.analytics.constants.ScreenNames
 import org.p2p.wallet.common.mvp.BasePresenter
 import org.p2p.wallet.utils.emptyString
+import timber.log.Timber
 
 private const val VIBRATE_DURATION = 500L
 
+enum class PinMode {
+    CREATE,
+    CONFIRM
+}
```

Comment:
Are you sure it's the right place for that class? 
I would move as a separate file

Reply:
Ok, let's separate it